x/ecocredit/server/testsuite/suite.go

```diff
@@ -973,14 +973,60 @@ func (s *IntegrationTestSuite) TestScenario() {
 		})
 	}
 
+	// reset the space to avoid corrupting other tests
+	s.paramSpace.Set(s.sdkCtx, ecocredit.KeyCreditTypes, ecocredit.DefaultParams().CreditTypes)
+
 	// TODO: test create sell order #627
 
+	askPrice := sdk.NewInt64Coin("stake", 1)
+
+	expectedSellOrderIds := []uint64{1, 2}
+
+	createSellOrder, err := s.msgClient.Sell(s.ctx, &ecocredit.MsgSell{
+		Owner:  addr3,
+		Orders: []*ecocredit.MsgSell_Order{
+			{
+				BatchDenom:        batchDenom,
+				Quantity:          "1.0",
+				AskPrice:          &askPrice,
+				DisableAutoRetire: true,
+			},
+			{
+				BatchDenom:        batchDenom,
+				Quantity:          "1.0",
+				AskPrice:          &askPrice,
+				DisableAutoRetire: true,
+			},
+		},
+	})
+	s.Require().Nil(err)
+	s.Require().Equal(expectedSellOrderIds, createSellOrder.SellOrderIds)
```

Comment:
This is one valid case that creates a sell order allowing us to query a sell order by ID. The integration tests will still need to be completed (#627).